README.md

```diff
@@ -32,13 +32,15 @@ Start by reaching out on the community channels (Reddit, Discord, IRC, etc.
 see the [Community Channels](http://docs.godotengine.org/en/stable/community/channels.html) doc),
 then create your proposal once you have gained some interest.
 
-5. You can make a PR implementing the feature in the main repository before 
+5. If a post receives little or no engagement after 30 days it will be closed.
```

Comment:
```suggestion
5. If a post receives little or no engagement after 90 days it will be closed.
```
I'm still not sure whether this place could be used for proposals with extended discussions. There are quite a bunch of proposals that really end up as discussions which were opened like 2 years ago but still updated recently, [see here](https://github.com/godotengine/godot/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+proposal%22+sort%3Aupdated-desc).

Also, you wouldn't really want to close godotengine/godot#14011 at all. 😛

Reply:
That wouldn't be captured by this rule because there was engagement 18 hours ago. 

Discussion is engagement. We are talking about posts that have received no comments for 30 full days. 

Additionally they are closed at our discretion, if there has been. A lot of productive discussion then we can just leave it open.

Reply:
Discussion can continue on a closed issue. Maybe we can add a rule to re-open closed issues if relevant discussion is active again.

Reply:
@xrayex I didn't add your 90 day time limit, but I did add @vnen's rule about reopening closed proposals. whether the limit is 30 days or 90, there will be posts that slip through the cracks and get closed. Having a rule to reopen them should be enough to ensure that they get reopened.